chef/cookbooks/neutron/templates/default/ovs_neutron_plugin.ini.erb

```diff
@@ -64,7 +64,7 @@
 # the server.  In most cases, the default value should be fine.
 #
 # tunnel_bridge = br-tun
-<% if @ml2_type_drivers.include?("gre") -%>
+<% if @ml2_type_drivers.include?("gre") || @ml2_type_drivers.include?("vxlan") -%>
```

Comment:
IIRC from a conversation a long time ago, that's the same crazy brain which thinks that

    "... \"foo bar\" \"baz qux\" ..."

    is more readable than

    '... "foo bar" "baz qux" ...'

;-p  Just goes to show that personal preferences can be very different.  But I also don't mind either way (as long as no more copy'n'paste happens in the future) so I'll leave it to you to choose ;-)